Makefile

```diff
@@ -50,30 +50,49 @@ endif
 endif
 export BASEIMAGE
 
+ifeq (${IMAGE_REPO}, )
```

Comment:
Comments will be useful about what `IMAGE_REPO` means. So users won't just put `quay.io` instead of `quay.io/openebs`

Reply:
This actually should be IMAGE_ORG. Updated and added comments. Good catch!